.vscode/settings.json

```diff
@@ -4,12 +4,25 @@
     "SetSource",
     "DefineClass",
     "DefineAbility",
+    "DefineAbilityCategory",
+    "DefineDomain",
+    "DefineStat",
+    "DefineSave",
+    "DefineFact",
+    "DefineVariable",
+    "DefineAlignment",
     "ModifyAbility",
+    "ModifyDomain",
+    "ChooseAbilitySelection",
     "ChooseSkill",
-    "ChooseWeaponproficiency",
+    "ChooseClass",
+    "ChooseLanguage",
+    "ChooseWeaponProficiency",
     "ChooseString",
+    "ChooseUserInput",
     "ChooseSchool",
     "ChooseSpell",
+    "ChooseNothing",
```

Comment:
ChooseNothing isn't super interesting to write down, since it's just a hack in the data files, but it's easy enough to just ignore. :-)